src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.html

```diff
@@ -69,8 +69,20 @@
                     i18n-placeholder
                     id="token"
                     formControlName="token"
-                    readonly>
-          </textarea>
+                    readonly
+                    #token></textarea>
+        </div>
+
+        <div class="btn-group mb-3"
+             role="group"
+             aria-label="Copy To Clipboard">
+          <button class="btn btn-primary"
+                  title="Copy to Clipboard"
+                  i18n-title
+                  (click)="copyToClipBoard(token)"> Copy to Clipboard
+            <i class="fa fa-files-o"
+               aria-hidden="true"></i>
+          </button>
         </div>
```

Comment:
I would suggest the following changes:
 * Since this is a single button, we don't really need to put it inside a btn-group or an extra div.
 * I think, we don't have any button align to the left, so I would align it to the right.
 * Missing I18N for the button text.

```suggestion
        <button class="btn btn-primary mb-3 float-right"
                title="Copy to Clipboard"
                i18n-title
                aria-label="Copy To Clipboard"
                (click)="copyToClipBoard(token)">
          <ng-container i18n>Copy to Clipboard</ng-container>
          <i class="fa fa-files-o"
             aria-hidden="true"></i>
        </button>
```

 I also notice some cleanup could be done to this component, so I create a new tracker issue:
https://tracker.ceph.com/issues/44848

Reply:
Done